scripts/build.ps1

```diff
@@ -30,6 +30,9 @@ This script provides helpers for building msquic.
 .PARAMETER DisableTest
     Don't build the test directory.
 
+.PARAMETER DisablePerf
+    Don't build the perf directory.
```

Comment:
The perf pipeline should now be able to disable test and tool code from building.